src/components/Input.js

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+function Input({value, onChange, placeholderText}) {
+
+    return (
+        <>
+            <InputStyled 
+                value={value} 
+                onChange={onChange} 
+                placeholder={placeholderText}>
+            </InputStyled>
+        </>
+ )
+}
+
+const InputStyled = styled.input`
```

Comment:
😑✋  StyledInput
😏👉 InputStyled

Reply:
Actually, we just talked about this on the team, in this Case neither seems the best choice. `InputStyled` would be more correct if you style an existing component called `Input` (i.e. `styled(Input)`).

The biggest factor here is that this is basically a redundant React component, and this should have been my initial reviewing comment for this. The best choice here would be removing all React related things entirely and returning the styled component as default export, because you are basically just passing props on to the styled component.

Cheers ;)